package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
+      "function-parentheses-space-inside": "always",
+      "function-url-quotes": "always",
```

Comment:
Correct. Sometimes not using quotes can cause issues with inlined images for example, so it's a good habit to have.